File: buzzvil-sdk-ios/buzzvil-sdk-ios-objc/Feed/Custom/CustomFeedEntryView.m
#import "CustomFeedEntryView.h"

@interface CustomFeedEntryView ()

@property (nonatomic, strong, readonly) UIView *customSubview1;
@property (nonatomic, strong, readonly) UIView *customSubview2;

@end

@implementation CustomFeedEntryView

- (instancetype)initWithCoder:(NSCoder *)coder {
  if (self = [super initWithCoder:coder]) {
    [self setupView];
  }
  return self;
}

- (instancetype)initWithFrame:(CGRect)frame {
  if (self = [super initWithFrame:frame]) {
    [self setupView];
  }
  return self;
}

- (void)setupView {
  // subview 초기화 및 LayoutConstraint 설정
  // ...생략...
  
  // 클릭이 가능한 영역을 지정합니다.
  self.clickableViews = @[self.customSubview1, self.customSubview2];
}

@end

File: buzzvil-sdk-v5-ios/buzzvil-sdk-ios-swiftui/Feed/Custom/CustomFeedEntryView.swift
import UIKit
import BuzzvilSDK

final class CustomFeedEntryView: BZVFeedEntryView {
  let customSubview1 = UIView(frame: .zero)
  let customSubview2 = UIView(frame: .zero)
  
  required init?(coder: NSCoder) {
    super.init(coder: coder)
    setupView()
  }
  
  override init(frame: CGRect) {
    super.init(frame: frame)
    setupView()
  }
  
  private func setupView() {
    // subview 초기화 및 LayoutConstraint 설정
    // ...생략...
    
    // 클릭이 가능한 영역을 지정합니다.
    clickableViews = [customSubview1, customSubview2]
  }
}
